Tests/ContentfulSyncSerializerTests/ContentfulSyncSerializerTests.swift

```diff
@@ -41,8 +41,8 @@ class ContentfulSyncSerializerTests: XCTestCase {
                     return
                 }
                 XCTAssertNotNil(try? testFolder.file(named: "locales.json"))
-                XCTAssertNotNil(try? testFolder.file(named: "0.json"))
-                XCTAssertNotNil(try? testFolder.file(named: "1.json"))
+                XCTAssertNotNil(try? testFolder.file(named: "entries.json"))
+//                XCTAssertNotNil(try? testFolder.file(named: "1.json"))
```

Comment:
Why is this commented out?

Reply:
because the file was renamed to something more meaningful (entries.json added under 1.json).